.github/workflows/build.yml

```diff
@@ -85,7 +85,7 @@ jobs:
       - name: Build CPython
         run: |
           make -j4 regen-all
-          make regen-stdlib-module-names regen-sbom
+          make regen-stdlib-module-names regen-sbom regen-jit
```

Comment:
I don't think this is needed (though I don't think it has any effect, since `regen-jit` is an empty target unless `--experimental-jit` is given).

Reply:
FTR, see `.github/workflows/jit.yml` for the JIT CI.

Reply:
Good point